_posts/2020-04-16-remote-junior-developer-tips.markdown

```diff
@@ -0,0 +1,57 @@
+
+### Pivotal Tracker:
+We use pivotal tracker to keep track of user stories, and set sprints for the week. Pivotal Tracker also allows for comments and this has become a useful tool when keeping track of what’s been worked on in a story with a Senior Developer.
+
+You can explain the progress that has been made on a story so far. Then, when you  need to ask a Senior Developer for help, you can point them directly to the story which already holds information on the problem.
+
+### Pairing:
+Pairing is the most important tool available to a remote Junior Developer. At OmbuLabs we have a ratio of six Senior Developers to one Junior Developer at the moment. This has allowed for a robust paring schedule. Each senior developer gives two hours of time every other week, that way the Junior Developer receives a total of 6 pairing hours a week. This is invaluable time in the learning process.
+
+## Tracking Progress
+
+On the journey to becoming a Developer there is A LOT to learn. Even when you finally make it to a comfortable place in your Developer career there will still be so much more to learn, so keeping track of it all is very helpful in the process.
+
+### Journal:
+Keeping a journal (I use a google doc) is an important step in organizing all the things you learn or learn that you should learn. During pair sessions and throughout the day it is a good idea to take notes and jot down various resources and ideas that are thrown your way.
+
+### Backlog Grooming:
+This is a term stolen from agile, but it also applies well when looking back at the information and projects you’ve been gathering. It’s a good idea to go through the journal or notes, pull out things you need to study or work on, and check off ideas or concepts that you have gained an understanding in.
+
+## Getting to know the team
+
+When working in a remote environment it is difficult to form relationships and get to know your teammates, this is very important for the Junior Developer though because it helps you communicate better about work. A Junior Developer is much more likely to communicate effectively if they feel comfortable with the Senior Developer on the other end of the line. At OmbuLabs we have several tools that help to build relationships.
+
+### Retreat:
+This year we held a team retreat, this has made all the difference in company culture. As the Junior Developer it helped me build relationships with Senior Developers so that feelings of awkwardness and nervousness began to dissipate. You can read more about our retreat [here](https://www.ombulabs.com/blog/agile/remote/team-retreat-for-remote-companies.html).
+
+### A Get To Know You Chat:
+Instrumental in the onboarding process at OmbuLabs is the chance to set up half hour meetings with each of your colleagues to learn a little more about them as people, and what they specifically do within the company.
```

Comment:
@FionaDL A link to https://www.ombulabs.com/blog/slack/scrum/agile/onboarding-with-slack-workflows.html would be nice

```suggestion
Instrumental in the [onboarding process](https://www.ombulabs.com/blog/slack/scrum/agile/onboarding-with-slack-workflows.html) at OmbuLabs is the chance to set up half hour meetings with each of your colleagues to learn a little more about them as people, and what they specifically do within the company.
```